nixos/modules/programs/gnupg.nix

```diff
@@ -6,6 +6,20 @@ let
 
   cfg = config.programs.gnupg;
 
+  xserverCfg = config.services.xserver;
+
+  defaultPinentryFlavour =
+    if xserverCfg.desktopManager.gnome3.enable
+    || xserverCfg.desktopManager.pantheon.enable then
+      "gnome3"
```

Comment:
Umm, we have xfce and xfce4-14 as well.

Reply:
The old xfce probably should use gtk2

Reply:
In that case, the "xserver" default (`gtk2`) should kick in, shouldn't it?

Reply:
Yep, but we still need it for `xserverCfg.desktopManager.xfce4-14.enable` since that uses gtk3 now. So we need an `||` for it with `gnome3` here.